packages/account-v2/src/pages/LoginHistory/LoginHistory.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Table } from '@deriv-com/ui';
+
+const headers = ['Date and Time', 'Action', 'Browser', 'IP Address', 'Status'] as const;
+const columns = headers.map(header => ({
+    header,
+}));
```

Comment:
why do we need this?

Reply:
In order to define the headers of the table we have pre defined it where headers contains the contents and column maps each header and displays it individually in  columns 
this segmentation is what was used in deriv-ui as well refering to the fromat used there and suggestions in the above comments i have written this